test/e2e/telemetry_logs_analysis_test.go

```diff
@@ -25,117 +24,128 @@ import (
 	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
 )
 
-var _ = Describe(suite.ID(), Label(suite.LabelTelemetryLogsAnalysis), Ordered, func() {
+var _ = Describe(suite.ID(), Label(suite.LabelTelemetryLogAnalysis), Ordered, func() {
 	const (
-		otelCollectorNs             = "tlogs-otelcollector"
-		fluentBitNs                 = "tlogs-fluentbit"
-		otelCollectorLogBackendName = "tlogs-otelcollector-log"
-		fluentBitLogBackendName     = "tlogs-fluentbit-log"
-		metricBackendName           = "tlogs-metric"
-		traceBackendName            = "tlogs-trace"
-		pushMetricsDepName          = "push-metrics-istiofied"
 		consistentlyTimeout         = time.Second * 120
+		traceBackendName            = "trace-backend"
+		metricBackendName           = "metric-backend"
+		logBackendName              = "log-backend"
+		otelCollectorLogBackendName = "otel-collector-log-backend"
+		fluentBitLogBackendName     = "fluent-bit-log-backend"
+		selfMonitorLogBackendName   = "self-monitor-log-backend"
 	)
 
 	var (
-		otelCollectorLogPipelineName     string
-		fluentBitLogPipelineName         string
-		metricPipelineName               string
-		tracePipelineName                string
-		otelCollectorLogbackendExportURL string
-		fluentBitLogbackendExportURL     string
-		metricbackendExportURL           string
-		tracebackendExportURL            string
-		gomegaMaxLength                  = format.MaxLength
-		errorWarningLevels               = []string{
-			"ERROR", "error",
-			"WARNING", "warning",
-			"WARN", "warn"}
+		traceBackendURL            string
+		metricBackendURL           string
+		logBackendURL              string
+		otelCollectorLogBackendURL string
+		fluentBitLogBackendURL     string
+		selfMonitorLogBackendURL   string
+		namespace                  = suite.ID()
```

Comment:
I would still recommend separating them into 3 namespaces. You can use the `suite.IDWithSuffix("...")` function

Reply:
You would separate the backends into 3 namespaces? I changed the approach to have one context only, so having 3 namespaces brings no benefit in my understanding? Why you think so?

Reply:
I was just thinking of separating the 3 scenarios, so they don't influence each other in any way. But you are right, I think with your current implementation you won't be able to properly gather the logs if you split it in namespaces. Leave it like this then.